modules/fluid_properties/doc/content/source/userobjects/SinglePhaseFluidProperties.md

```diff
@@ -1,8 +1,83 @@
 # SinglePhaseFluidProperties
 
-`SinglePhaseFluidProperties` is a base class for all single phase fluid properties.
+`SinglePhaseFluidProperties` is a base class for all single-phase fluid properties
+objects. Its main function is to provide interfaces for computing various properties
+from different combinations of other properties.
 
-## Default analytical fluid properties relations
+## Properties
+
+The following properties are considered in this class, where the "Name" column
+gives the identifier for the property used in the available interfaces:
+
+| Name | Symbol | Description |
+| :- | :- | :- |
+| `beta` | $\beta$ | Volumetric expansion coefficient |
+| `c` | $c$ | Speed of sound |
+| `cp` | $c_p$ | Isobaric specific heat capacity |
+| `cv` | $c_v$ | Isochoric specific heat capacity |
+| `e` | $e$ | Specific internal energy |
+| `g` | $g$ | Gibbs free energy |
+| `gamma` | $\gamma$ | Ratio of specific heats, $\frac{c_p}{c_v}$ |
+| `h` | $h$ | Specific enthalpy |
+| `k` | $k$ | Thermal conductivity |
+| `mu` | $\mu$ | Dynamic viscosity |
+| `p` | $p$ | Pressure |
+| `rho` | $\rho$ | Density |
+| `s` | $s$ | Specific entropy |
+| `T` | $T$ | Temperature |
+| `v` | $v$ | Specific volume |
+
+Because two independent, intensive thermodynamic properties define a
+thermodynamic state of a pure fluid, interfaces in these objects are of the form $f(a,b)$
+where $f$ is the desired thermodynamic property and $a$ and $b$ are independent,
+intensive thermodynamic properties that define the thermodynamic state. The
+corresponding function name is `fname_from_aname_bname`, where `fname`, `aname`,
+and `bname` are the names in the table above, corresponding to $f$, $a$, and $b$,
+respectively. The following table lists which
+properties are available from various combinations of properties (e.g., "Yes"
+in the column $(a,b)$ for the row $f$ denotes that the interface `fname_from_aname_bname`
+is available):
+
+| Name     | $(p,T)$ | $(v,e)$ | $(p,s)$ | $(p,h)$ | $(T,v)$ | $(v,h)$ | $(p,\rho)$ |
+| :-       | -       | -       | -       | -       | -       | -       | -          |
+| $\beta$  | Yes     |         |         |         |         |         |            |
+| $c$      | Yes     | Yes     |         |         |         |         |            |
+| $c_p$    | Yes     | Yes     |         |         |         |         |            |
+| $c_v$    | Yes     | Yes     |         |         | Yes     |         |            |
+| $e$      | Yes     |         |         |         | Yes     | Yes     | Yes        |
+| $g$      |         | Yes     |         |         |         |         |            |
+| $\gamma$ | Yes     |         |         |         |         |         |            |
+| $h$      | Yes     |         |         |         | Yes     |         |            |
+| $k$      | Yes     | Yes     |         |         |         |         |            |
+| $\mu$    | Yes     | Yes     |         |         |         |         |            |
```

Comment:
mu(rho, T) as well

Reply:
Thanks for finding all of these missing ones (and the extra one)!